src/prefect/orion/database/migrations/versions/postgresql/2022_08_18_102804_2d5e000696f1_adds_block_schema_version.py

```diff
@@ -0,0 +1,43 @@
+"""Adds block schema version
+
+Revision ID: 2d5e000696f1
+Revises: 77eb737fc759
+Create Date: 2022-08-18 10:28:04.449256
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "2d5e000696f1"
+down_revision = "77eb737fc759"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "block_schema",
+        sa.Column(
+            "version", sa.String(), server_default="non-versioned", nullable=False
+        ),
+    )
+    op.drop_index("ix_block_schema__capabilities", table_name="block_schema")
```

Comment:
Is this not used anywhere?

Reply:
Good eye! I missed that in alembic's auto-generated migration. Fixed in https://github.com/PrefectHQ/prefect/pull/6491/commits/17f719e923e330cf8d73ff6d2ae084558ebba133.